src/BinSkim.Driver.FunctionalTests/BaselineTestsData/NonWindowsExpected/gcc.shared_library.so.sarif

```diff
@@ -0,0 +1,610 @@
+        {
+          "ruleId": "BA3002",
+          "level": "pass",
+          "formattedRuleMessage": {
+            "formatId": "Pass",
+            "arguments": [
+              "gcc.shared_library.so"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file://Z:/src/BinSkim.Driver.FunctionalTests/BaselineTestsData/gcc.shared_library.so"
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA3001",
+          "level": "pass",
+          "formattedRuleMessage": {
+            "formatId": "Pass_Library",
+            "arguments": [
+              "gcc.shared_library.so"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file://Z:/src/BinSkim.Driver.FunctionalTests/BaselineTestsData/gcc.shared_library.so"
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA3010",
+          "level": "pass",
+          "formattedRuleMessage": {
+            "formatId": "Pass",
+            "arguments": [
+              "gcc.shared_library.so"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file://Z:/src/BinSkim.Driver.FunctionalTests/BaselineTestsData/gcc.shared_library.so"
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA3003",
+          "level": "pass",
+          "formattedRuleMessage": {
+            "formatId": "Pass",
+            "arguments": [
+              "gcc.shared_library.so"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file://Z:/src/BinSkim.Driver.FunctionalTests/BaselineTestsData/gcc.shared_library.so"
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA3030",
+          "level": "error",
+          "formattedRuleMessage": {
+            "formatId": "Error",
+            "arguments": [
+              "gcc.shared_library.so"
+            ]
+          },
+          "locations": [
+            {
+              "analysisTarget": {
+                "uri": "file://Z:/src/BinSkim.Driver.FunctionalTests/BaselineTestsData/gcc.shared_library.so"
+              }
+            }
+          ]
+        }
+      ],
+      "configurationNotifications": [
+        {
+          "id": "WRN998.NotSupportedPlatform",
+          "physicalLocation": {},
+          "message": "Rule 'BuildWithSecureTools' was disabled as it cannot run on the current platform 'Windows'.  It can only run on 'Linux'.",
+          "level": "note"
+        },
+        {
+          "id": "WRN998.NotSupportedPlatform",
+          "physicalLocation": {},
+          "message": "Rule 'DoNotDisableStackProtectionForFunctions' was disabled as it cannot run on the current platform 'Windows'.  It can only run on 'Linux'.",
```

Comment:
OS comments are backwards due to https://github.com/Microsoft/sarif-sdk/pull/767